src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int RANDOM_BOUND = Color.values().length;
+    private Random randomColor = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = randomColor.nextInt(RANDOM_BOUND);
+        return Color.values()[index].toString();
```

Comment:
Instead of using `toString()`, you should use `name()` for getting String representation of enum constants to avoid issues if `toString()` is overridden in the enum.